.github/workflows/ci-cd.yml

```diff
@@ -88,6 +88,15 @@ jobs:
         COLOR: 'yes'
       run: |
         uv run make mototest
+    # TODO: I don't know how to do this properly with uv-managed venv
```

Comment:
You could move `httpx` from dependency group `dev` into a new one, such as `httpx`, and also make that dependency group a default one. Then it would be as simple as `uv run --no-group httpx`.